cordova-lib/src/cordova/util.js

```diff
@@ -297,7 +297,7 @@ function ensurePlatformOptionsCompatible (platformOptions) {
     }).map(function (arg) {
         return opts[arg] === true ?
             '--' + arg :
-            '--' + arg + '="' + opts[arg].toString() + '"';
+            '--' + arg + '=' + opts[arg].toString();
```

Comment:
Why are we passing string CLI arguments in a programmatic API? If this is because we're using `cordova.raw`, then can we refactor to not use it?